BookAPI/Models/Book.cs

```diff
@@ -72,5 +72,38 @@ public Book(string title, string author, int numberOfPages, int year)
         /// </value>
         [Range(-2000, 2018, ErrorMessage = "Wrong year parameter (never heard about books from that time)!")]
```

Comment:
What will you do in 2019? Will you open your application and change 2018 to 2019, and so on every year?))

Reply:
And one more trick: if you use such range, and will try to send book from UI to controller with no specified year, the year of this book will get the value 0 and will not show you any mistake